test/stac/stac.ts

```diff
@@ -160,4 +162,63 @@ describe('STAC catalog route', function () {
       });
     });
   });
+
+  describe('For a guest user who does not own the job', function () {
+    describe('when the job is incomplete', function () {
+      hookStacCatalog(jobId);
+      it('returns an HTTP not implemented response', function () {
+        expect(this.res.statusCode).to.equal(409);
+      });
+
+      it('returns a JSON error response', function () {
+        const response = JSON.parse(this.res.text);
+        expect(response).to.eql({
+          code: 'harmony.ConflictError',
+          description: `Error: Job ${jobId} is not complete`,
+        });
+      });
+    });
+
+    describe('when the job is complete', function () {
+      describe('when the service supplies the necessary fields', async function () {
+        const completedJobId = completedJob.requestId;
+        hookStacCatalog(completedJobId);
+
+        it('returns an HTTP OK response', function () {
+          expect(this.res.statusCode).to.equal(200);
+        });
+
+        it('returns a STAC catalog in JSON format', function () {
+          const reqUrl = new url.URL(this.res.request.url);
+          const catalog = JSON.parse(this.res.text);
+          expect(catalog.description).to.equal('Harmony output for http://example.com/harmony?job=completedJob');
+          expect(catalog.id).to.equal(completedJob.requestId);
+          expect(catalog.links).to.eql([
+            { href: '.', rel: 'root', title: 'root' },
+            { href: './0', rel: 'item' },
+            {
+              href: `${reqUrl.origin}/stac/${completedJob.requestId}?page=1&limit=${env.defaultResultPageSize}`,
+              rel: 'self',
+              title: 'The current page',
+              type: 'application/json',
+            },
+          ]);
+          expect(catalog.stac_version).to.equal('1.0.0');
+          expect(catalog.title).to.include('Harmony output for ');
+          expect(catalog.type).to.equal('Catalog');
+        });
+      });
+      describe('when the linkType is invalid', function () {
```

Comment:
Similar comment - this part doesn't seem related to the user type permissions and looks like it is already tested elsewhere.